### PR TITLE
Added tutorials

### PR DESCRIPTION
**Pull request by ClarytyLLC**
Created a new branch and added tutorials to resolve conflicts.

### CODE REVIEW

File: src/docs/tutorials/pages/customizing-engine-input.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 **APPROXIMATE READING TIME: 7 MINUTES**
 
->**Tip** Before getting started, we recommend reading [Build Your Own Engine](/developer/engines/tutorial/) to understand how you can build your engine. We recommend you also read the [Customize Engine Output](customizing-engine-output) tutorial.
+>**Tip** Before getting started, we recommend to read  [Build Your Own Engine](/developer/engines/tutorial/) to understand how you can build your engine. We recommend you also read the [Customize Engine Output](customizing-engine-output) tutorial.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-input.md:5]: replace "to read" with "reading"

File: src/docs/tutorials/pages/customizing-engine-input.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Let us say we are registering a new engine with Veritone. At the bottom of the f
 
 In the above example, we have added a custom field with name `outputLanguage` and label Output Language. When we run our cognitive engine on the [Veritone CMS](https://cms.veritone.com/), the UI prompts us to specify a value for Output Language. The value specified will be passed as  `outputLanguage` form-field parameter in the body and is POSTed to the engine's `/process` handler at runtime.
 
-Next, we will run the engine and view the output.
+Next, we will run run the engine and view the output.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-input.md:101]: remove the duplicate "run"

File: src/docs/tutorials/pages/customizing-engine-output.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 **APPROXIMATE READING TIME: 8 MINUTES**
 
->**Tip** Before getting started, we recommend reading  [Build Your Own Engine](/developer/engines/tutorial/) to understand how you can build your engine and generate an output.
+>**Tip** Before getting started, we recommend to read  [Build Your Own Engine](/developer/engines/tutorial/) to understand how you can build your engine and generate an output.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:5]: replace "to read" with "reading"

File: src/docs/tutorials/pages/customizing-engine-output.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ We will first create a summary schema and include it in our output. We will be c
 
 The sample `stats` object has three fields, all of type integer. However, your summary object can include any number of fields of any data type supported by JavaScript.
 
-In order to use this data structure in our output, we will need to create the JSON schema that supports it. You could use any of the easy-to-use online tools, such as  https://jsonschema.net/, to create a schema. A schema that supports the above data structure looks like:
+In order to use a this data structure in our output, we will need to create the JSON schema that support it. You could use any of the easy-to-use online tools, such as  https://jsonschema.net/, to create a schema. A schema that supports the above data structure looks like:
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:51]: remove "a"

Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:51]: Replace "support" with "supports"

**Comment by myoungerman** [comment]
@ClarytyLLC Besides sidebar.md and the entirely new file you added, please revert all of the grammatical changes to what they were before.

File: src/docs/tutorials/pages/customizing-engine-output.md (hunk 4 above)
Comment (ClarytyLLC) [reply src/docs/tutorials/pages/customizing-engine-output.md:51]: Updated

Comment (ClarytyLLC) [reply src/docs/tutorials/pages/customizing-engine-output.md:51]: Updated

File: src/docs/tutorials/pages/customizing-engine-input.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ In this tutorial we will learn together how to pass a custom input to your engin
 - Specify the URL of a web hook.
 - Point to a schema.
 
-These are just a few examples. There could be many other possible scenarios where you would use custom parameters.  You can easily pass an arbitrary JSON data to an engine in aiWARE. Let us quickly go through how it is done.
+These are just a few examples. There could be many other possible scenarios where you would use custom parameters.  You can easily pass an arbitary JSON data to an engine in aiWARE. Let us quickly go through how it is done.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-input.md:17]: Change "arbitary" back to "arbitrary"

File: src/docs/tutorials/pages/customizing-engine-output.md (hunk 3 above)
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:5]: Change "to read" back to "reading"

File: src/docs/tutorials/pages/customizing-engine-output.md [hunk 6]
```diff
@@ -140,7 +140,7 @@ In the above example, we have customized some of the titles just to suit our nee
 
    
 
-7. Click the three-vertical-dots ("kebab") menu on the right in the row for your schema, and select **PUBLISH**. This publishes our schema and makes it available. Next, we need to get the schema Id for our schema.
+7. Click the three-vertical-dots ("kebab") menu on the right in the row for your schema, and select **PUBLISH**. This publishes our schema and makes it available. next, we need to get the schema Id for our schema.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:143]: Capitalize "next"

File: src/docs/tutorials/pages/customizing-engine-output.md [hunk 7]
```diff
@@ -215,7 +215,7 @@ let _words = null;
 
 These lines support our timing variable, `_startTime`, and a new global `_words` (for word count), in this snippet.
 
-You can save the file, rebuild the engine, and test it locally (using Test Mode) as described in the [Build Your Own Engine](#/developer/engines/tutorial/) tutorial. After you verify that the engine works as planned, you can deploy it to Veritone.
+You can Save the file, re-build the engine, and test it locally (using Test Mode) as described in the [Build Your Own Engine](#/developer/engines/tutorial/) tutorial. After you verify that the engine works as planned, you can deploy it to Veritone.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-output.md:218]: Make "Save" lowercase

File: src/docs/tutorials/pages/polling.md [hunk 8]
```diff
@@ -2,11 +2,11 @@
 
 **APPROXIMATE READING TIME: 4 MINUTES**
 
->**Tip** Before getting started, we recommend reading [Training an Engine](training-an-engine) and [Working with Jobs](#/developer/engines).
+>**Tip** Before getting started, we recommend to read about [Training an Engine](training-an-engine) and [Working with Jobs](#/developer/engines).
```
Comment (myoungerman) [inline src/docs/tutorials/pages/polling.md:5]: Change "to read about" back to "reading"

File: src/docs/tutorials/pages/polling.md [hunk 9]
```diff
@@ -2,11 +2,11 @@
...
 
 ## Polling a Job for Status <!-- {docsify-ignore} -->
 
-In this tutorial, we will learn together how to do polling. You can review the status of your job by polling its status. The possible statuses of a job are:
+In this tutorial, we will learn together how to do perform polling. You can review the status of your job by polling its status. The possible statuses of a job are:
```
Comment (myoungerman) [inline src/docs/tutorials/pages/polling.md:9]: Change "do perform polling" to "poll the status of a job"

File: src/docs/tutorials/pages/testing-and-debugging-engines.md [hunk 10]
```diff
@@ -0,0 +1,271 @@
+# Testing and Debugging Engines
+
+**APPROXIMATE READING TIME: 10 MINUTES**
+
+>**Tip** Before getting started, we recommend reading about [how to quickly get started with aiWARE](/getting-started/quickstart/) and get familiar with our [cognitive engines](/cognitive-technology/). It would also help if you read about [training an engine](training-an-engine) and [working with jobs](#/developer/engines).
+
+## Engine Validation and Debugging <!-- {docsify-ignore} -->
+
+In this tutorial, we will learn together how to test our engine and debug the most common error. You should validate your engine before you submit a build.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/testing-and-debugging-engines.md:9]: Change all instances of "we" to "you"

File: src/docs/tutorials/pages/customizing-engine-input.md [hunk 11]
```diff
@@ -14,7 +14,7 @@ In this tutorial we will learn together how to pass a custom input to your engin
 - Specify the URL of a web hook.
 - Point to a schema.
 
-These are just a few examples. There could be many other possible scenarios where you would use custom parameters.  You can easily pass an arbitary JSON data to an engine in aiWARE. Let us quickly go through how it is done.
+These are just a few examples. There could be many other possible scenarios where you would use custom parameters.  You can easily pass an arbitrary JSON data to an engine in aiWARE. Let us quickly go through how it is done.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/customizing-engine-input.md]: Delete 'an'

File: src/docs/tutorials/pages/testing-and-debugging-engines.md [hunk 12]
```diff
@@ -2,11 +2,11 @@
 
 **APPROXIMATE READING TIME: 10 MINUTES**
 
->**Tip** Before getting started, we recommend reading about [how to quickly get started with aiWARE](/getting-started/quickstart/) and get familiar with our [cognitive engines](/cognitive-technology/). It would also help if you read about [training an engine](training-an-engine) and [working with jobs](#/developer/engines).
+>**Tip** Before getting started, you recommend reading about [how to quickly get started with aiWARE](/getting-started/quickstart/) and get familiar with our [cognitive engines](/cognitive-technology/). It would also help if you read about [training an engine](training-an-engine) and [working with jobs](#/developer/engines).
 
 ## Engine Validation and Debugging <!-- {docsify-ignore} -->
 
-In this tutorial, we will learn together how to test our engine and debug the most common error. You should validate your engine before you submit a build.
+In this tutorial, you will learn together how to test our engine and debug the most common error. You should validate your engine before you submit a build.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/testing-and-debugging-engines.md:9]: delete 'together'

File: src/docs/tutorials/pages/testing-and-debugging-engines.md [hunk 13]
```diff
@@ -27,7 +27,7 @@ Before doing functional testing of your engine, you should take care to see that
 
 ### Engine Output Validation API (Beta)
 
-We understand that writing valid engine output for the first time can be difficult and confusing.
+you understand that writing valid engine output for the first time can be difficult and confusing.
 To make it easier to write conforming output, Veritone provides a simple HTTP POST endpoint that you can use to test the validity of your engine's output.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/testing-and-debugging-engines.md:31]: replace 'conforming' with 'valid engine'

File: src/docs/tutorials/pages/testing-and-debugging-engines.md [hunk 14]
```diff
@@ -27,7 +27,7 @@ Before doing functional testing of your engine, you should take care to see that
 
 ### Engine Output Validation API (Beta)
 
-We understand that writing valid engine output for the first time can be difficult and confusing.
+you understand that writing valid engine output for the first time can be difficult and confusing.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/testing-and-debugging-engines.md:30]: delete line 30

File: src/docs/tutorials/pages/testing-and-debugging-engines.md [hunk 15]
```diff
@@ -83,7 +83,7 @@ curl --request POST \
 }'
 ```
 
-If the engine output is valid, you will see `valid: true` under the `data` key of the response, as well as a `processed` field, which will show you only the fields that have been validated against the vtn-standard.
+If the engine output is valid, you will see `valid: true` under the `data` key of the response, as youll as a `processed` field, which will show you only the fields that have been validated against the vtn-standard.
```
Comment (myoungerman) [inline src/docs/tutorials/pages/testing-and-debugging-engines.md:86]: replace 'youll' with 'well'

**Review by myoungerman — APPROVED** [review]

**Pull request merged**